new/LIT7072QeneMamhera.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7072QeneMamhera" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mamǝhǝra zǝntu ʿālam, ʾAwǝsṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro… (Qǝne of unknown type)</title>
```

Comment:
not Mamhǝra ?

Reply:
It's a question of contention, I think! Etymologically, one would expect mamǝhǝr (cf. magaśśǝṣ), but in the current pronunciation it's without schwa

Reply:
I think, we should be consistent to other Bm records. Some work records (13), many PRS records (35) and very many manuscript records (217) contain the transcription mamhər. For this reason, we should use this transcription, even though it might be etymologically incorrect.

Reply:
If it has eight verses, I would define it as mawaddǝs type according to Guidi 1901.

Reply:
> I think, we should be consistent to other Bm records. Some work records (13), many PRS records (35) and very many manuscript records (217) contain the transcription mamhər. For this reason, we should use this transcription, even though it might be etymologically incorrect.

Can we add this to the list of dubious spellings? https://betamasaheft.eu/Guidelines/?id=dubious-spelling @eu-genia 

Or would it be preferable to have a bigger discussion about it before?

Reply:
> > I think, we should be consistent to other Bm records. Some work records (13), many PRS records (35) and very many manuscript records (217) contain the transcription mamhər. For this reason, we should use this transcription, even though it might be etymologically incorrect.
> 
> Can we add this to the list of dubious spellings? https://betamasaheft.eu/Guidelines/?id=dubious-spelling @eu-genia
> 
> Or would it be preferable to have a bigger discussion about it before?

the dubious spellings from the Guidelines are the result of discussion in TraCES.

at least @abausi should have a say; I agree with @CarstenHoffmannMarburg that we must be consistent; if the decision is to adhere to etymology I would then replace all occurences to `mamǝhǝr `

Reply:
I would say, we uniformly transliterate (not transcribe) is as "mamhǝr"... The word is in modern languages , that makes the matter complicated.

Reply:
> If it has eight verses, I would define it as mawaddǝs type according to Guidi 1901.

Ok! I add it.

Reply:
Please, no mamǝhǝr. It will be very odd.  It is mamhǝr since EAe, the pronunciation is mamhǝr (with a light phonetic schwa before -h) in living languages, and Leslau has it as mamhǝr. Mamǝhǝr in some records may result not from morphological consderations, but from mechanical transliteration.